module-log-parser/src/main/kotlin/org/quiltmc/community/cozy/modules/logs/processors/quilt/IncompatibleModProcessor.kt

```diff
@@ -23,7 +23,7 @@ import org.quiltmc.community.cozy.modules.logs.data.Order
 import org.quiltmc.community.cozy.modules.logs.types.LogProcessor
 import kotlin.time.Duration.Companion.minutes
 
-private const val THREAD_LINK = "https://forum.quiltmc.org/t/mod-incompatibility-megathread/261"
+private const val THREAD_LINK = "https://quiltmc.org/incompatible-mods.json"
```

Comment:
I mean if you're going to update the link you might as well also update the name of the variable but sure

Reply:
![image](https://github.com/QuiltMC/cozy-discord/assets/17211100/224109cc-8049-4c0f-8710-384f01a7b495)